Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
+
+RUN mkdir -p /src/app
+
+WORKDIR /src/app
+
+COPY . /src/app
+
+RUN yarn install
+
+EXPOSE 3000
+
+CMD ["npm", "run", "server-dev"]doc
```

Comment:
Is this supposed to be "CMD ["npm", "run", "server-dev"]doc" or "CMD ["npm", "run", "server-dev"]"?

Reply:
good catch thanks. what this doc is doing here... lol. Just removed in the commit now.